packages/typegpu/src/index.ts

```diff
@@ -2,15 +2,17 @@
  * @module typegpu
  */
 
-import { Storage, Uniform, Vertex } from './core/buffer/buffer';
 import { createBuffer } from './legacyBufferApi';
 import { bindGroupLayout } from './tgpuBindGroupLayout';
 import { read, write } from './tgpuBufferUtils';
 
 export const tgpu = {
-  Uniform,
-  Storage,
-  Vertex,
```

Comment:
If we're not exporting the types anymore as part of `tgpu`, we need to export them another way.

We can just add that to the last export line:
```ts
export type { TgpuBuffer, Uniform, Storage, Vertex } from './core/buffer/buffer';
```

This is so that, e.g., devs can properly type function arguments that accept buffers that have a specific usage.